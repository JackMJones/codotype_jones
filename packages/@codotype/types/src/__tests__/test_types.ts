import meaningOfLife from "../index";

test("testing @codotype/types", () => {
  expect(meaningOfLife).toBe(42);
});
