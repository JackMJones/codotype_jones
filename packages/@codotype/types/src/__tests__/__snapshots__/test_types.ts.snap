// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing @codotype/types 1`] = `
Object {
  "addons": Object {},
  "datatype": null,
  "defaultValue": null,
  "id": "",
  "identifiers": Object {
    "camel": "",
    "kebab": "",
    "label": "",
    "pascal": "",
    "snake": "",
  },
  "internalNote": "",
  "locked": false,
  "source": "USER",
}
`;

exports[`testing @codotype/types 2`] = `
Object {
  "destinationSchemaAlias": "",
  "destinationSchemaId": "",
  "id": "",
  "required": false,
  "source": "USER",
  "sourceSchemaAlias": "",
  "type": null,
}
`;

exports[`testing @codotype/types 3`] = `
Object {
  "attributes": Array [],
  "configuration": Object {},
  "id": "",
  "identifiers": Object {
    "plural": Object {
      "camel": "",
      "kebab": "",
      "label": "",
      "pascal": "",
      "snake": "",
    },
    "singular": Object {
      "camel": "",
      "kebab": "",
      "label": "",
      "pascal": "",
      "snake": "",
    },
  },
  "locked": false,
  "relations": Array [],
  "removable": true,
  "source": "USER",
}
`;
